src/test/ui/macros/unreachable-2021.rs

```diff
@@ -0,0 +1,9 @@
+// edition:2021
```

Comment:
Could you rename this test to something like `unreachable-2021-format_args.rs` to make it clear what it tests?

Reply:
And could you also merge this test with the `unreachable-2015` one using [revisions](https://rustc-dev-guide.rust-lang.org/tests/compiletest.html#revisions)?

Reply:
Done in https://github.com/rust-lang/rust/pull/93179/commits/d0e4240a1468b3b2e921324cbb7088850dacb4df